conftest.py

```diff
@@ -1,7 +1,41 @@
 import datetime
 
-post = {
-    'text': 'Everything you need to know to preorder Hogwarts Legacy, the upcoming open-world RPG set in '
-            'the wizarding world of Harry Potter, before the hero arrived.',
-    'photo': 'https://assets-prd.ignimgs.com/2022/03/18/hogwarts-legacy-4-1647622573681.jpeg?width=1280',
-    'time': datetime.datetime(2022, 3, 24, 16, 31, 9)}
+scraped_data = [{
+            'text': 'Welcome to the very first episode of Rogue Jam!\n\nOur judges are on the hunt for a game with '
+                    'Huge Potential, so join them as they tear through a Canadian wilderness rogue-like, '
+                    'a time-bending 2D shooter, and a hockey game with heart.',
+            'photo': 'http://s3.amazonaws.com/photos.sproutsocial.com/2022-04-14/2f1926ff-832a-4e26-81b7-ea8def6af7b4'
```

Comment:
Hmmm storing images in the form of a link is a problem I would say as the photo might get deleted